File: lottery-web/src/main/java/model/DoubleColorBall.java
package model;

import java.text.MessageFormat;

/**
 * Mapping table LOTQTDCB
 * 
 * @author E377712
 *
 */
public class DoubleColorBall {
	private String issueNumber;

	private String firstBall;

	private String secondBall;

	private String thirdBall;

	private String fourthBall;

	private String fifthBall;

	private String sixthBall;

	private String blueBall;

	/** yyyy-mm-dd */
	private String issueDateStr;

	public String getIssueNumber() {
		return issueNumber;
	}

	public void setIssueNumber(String issueNumber) {
		this.issueNumber = issueNumber;
	}

	public String getFirstBall() {
		return firstBall;
	}

	public void setFirstBall(String firstBall) {
		this.firstBall = firstBall;
	}

	public String getSecondBall() {
		return secondBall;
	}

	public void setSecondBall(String secondBall) {
		this.secondBall = secondBall;
	}

	public String getThirdBall() {
		return thirdBall;
	}

	public void setThirdBall(String thirdBall) {
		this.thirdBall = thirdBall;
	}

	public String getFourthBall() {
		return fourthBall;
	}

	public void setFourthBall(String fourthBall) {
		this.fourthBall = fourthBall;
	}

	public String getFifthBall() {
		return fifthBall;
	}

	public void setFifthBall(String fifthBall) {
		this.fifthBall = fifthBall;
	}

	public String getSixthBall() {
		return sixthBall;
	}

	public void setSixthBall(String sixthBall) {
		this.sixthBall = sixthBall;
	}

	public String getBlueBall() {
		return blueBall;
	}

	public void setBlueBall(String blueBall) {
		this.blueBall = blueBall;
	}

	public String getIssueDateStr() {
		return issueDateStr;
	}

	public void setIssueDateStr(String issueDateStr) {
		this.issueDateStr = issueDateStr;
	}

	@Override
	public String toString() {
		return MessageFormat.format("{0}    {1},{2},{3},{4},{5},{6}|{7}    {8}", this.issueNumber, this.firstBall,
				this.secondBall, this.thirdBall, this.fourthBall, this.fifthBall, this.sixthBall, this.blueBall,
				this.issueDateStr);
	}
}

File: lottery-php/DoubleColorBall.php
<?php
/**
 * Created by PhpStorm.
 * User: dxinfor
 * Date: 4/17/2017
 * Time: 11:38 AM
 */
class DoubleColorBall {
	private $issueNumber;
	private $firstBall;
	private $secondBall;
	private $thirdBall;
	private $fourthBall;
	private $fifthBall;
	private $sixthBall;
	private $blueBall;
	private $issueDateStr;
	
	/**
	 *
	 * @return mixed
	 */
	public function getIssueNumber() {
		return $this->issueNumber;
	}
	
	/**
	 *
	 * @param mixed $issueNumber        	
	 */
	public function setIssueNumber($issueNumber) {
		$this->issueNumber = $issueNumber;
	}
	
	/**
	 *
	 * @return mixed
	 */
	public function getFirstBall() {
		return $this->firstBall;
	}
	
	/**
	 *
	 * @param mixed $firstBall        	
	 */
	public function setFirstBall($firstBall) {
		$this->firstBall = $firstBall;
	}
	
	/**
	 *
	 * @return mixed
	 */
	public function getSecondBall() {
		return $this->secondBall;
	}
	
	/**
	 *
	 * @param mixed $secondBall        	
	 */
	public function setSecondBall($secondBall) {
		$this->secondBall = $secondBall;
	}
	
	/**
	 *
	 * @return mixed
	 */
	public function getThirdBall() {
		return $this->thirdBall;
	}
	
	/**
	 *
	 * @param mixed $thirdBall        	
	 */
	public function setThirdBall($thirdBall) {
		$this->thirdBall = $thirdBall;
	}
	
	/**
	 *
	 * @return mixed
	 */
	public function getFourthBall() {
		return $this->fourthBall;
	}
	
	/**
	 *
	 * @param mixed $fourthBall        	
	 */
	public function setFourthBall($fourthBall) {
		$this->fourthBall = $fourthBall;
	}
	
	/**
	 *
	 * @return mixed
	 */
	public function getFifthBall() {
		return $this->fifthBall;
	}
	
	/**
	 *
	 * @param mixed $fifthBall        	
	 */
	public function setFifthBall($fifthBall) {
		$this->fifthBall = $fifthBall;
	}
	
	/**
	 *
	 * @return mixed
	 */
	public function getSixthBall() {
		return $this->sixthBall;
	}
	
	/**
	 *
	 * @param mixed $sixthBall        	
	 */
	public function setSixthBall($sixthBall) {
		$this->sixthBall = $sixthBall;
	}
	
	/**
	 *
	 * @return mixed
	 */
	public function getBlueBall() {
		return $this->blueBall;
	}
	
	/**
	 *
	 * @param mixed $blueBall        	
	 */
	public function setBlueBall($blueBall) {
		$this->blueBall = $blueBall;
	}
	
	public function getIssueDateStr() {
		return $this->issueDateStr;
	}
	
	public function setIssueDateStr($issueDateStr) {
		$this->issueDateStr = $issueDateStr;
	}
}

?>